index.bs

```diff
@@ -1617,8 +1617,19 @@ that are returned to the caller when a new credential is created, or a new asser
 
     dictionary AuthenticatorAttestationResponseJSON {
         required Base64URLString clientDataJSON;
-        required Base64URLString attestationObject;
+        required Base64URLString authenticatorData;
         required sequence<DOMString> transports;
+        // The publicKey field will be missing if pubKeyCredParams was used to
+        // negotiate a public-key algorithm that the user agent doesn't
+        // understand. (See section “Easily accessing credential data” for a
+        // list of which algorithms user agents must support.) If using such an
+        // algorithm then the public key must be parsed from attestationObject
+        // directly.
```

Comment:
This can be `authenticatorData` too, right?
```suggestion
        // algorithm then the public key must be parsed directly from
        // attestationObject or authenticatorData.
```